package.json

```diff
@@ -49,7 +50,7 @@
     "tslint": "^4.0.0",
     "tslint-microsoft-contrib": "4.0.0",
     "typedoc": "^0.5.10",
-    "typescript": "^2.2.2"
+    "typescript": "2.3.0"
```

Comment:
I didn't. It looks like TS 2.3.1 has issues with tslint.

![](https://peet.io/i/17-06-20d53d21-5056-43dd-9d0a-357ab094211f.png)